index.html

```diff
@@ -141,6 +141,9 @@ <h3>File Sharing</h3>
                    be supported by servers as well as user agents.
    N15             It must be possible to support data exchange
                    in a worker.
+   N29             The application must be able to request user consent
```

Comment:
We should also consider allowing the exposure of local IP not via user prompt but in "out-of-band" methods like browser Enterprise policy